### PR TITLE
Support to create peering VPC 

### PR DESCRIPTION
**Pull request by haojinIntel**

### CODE REVIEW

File: python/cloudtik/providers/_private/aws/config.py [hunk 1]
```diff
@@ -1404,12 +1429,95 @@ def _delete_vpc_endpoint_for_s3(ec2_client, vpc_id, workspace_name):
...
+def _delete_routes_for_workspace_vpc_peering_connection(config, ec2, ec2_client):
+    workspace_name = config["workspace_name"]
+    current_vpc = get_current_vpc(ec2, config)
+    workspace_vpc = get_workspace_vpc(workspace_name, ec2_client, ec2)
+    current_vpc_route_tables = get_vpc_route_tables(current_vpc)
+    workspace_vpc_route_tables = get_vpc_route_tables(workspace_vpc)
+
+    for current_vpc_route_table in current_vpc_route_tables:
+        if len([route for route in current_vpc_route_table.routes if
+                route._destination_cidr_block == workspace_vpc.cidr_block]) == 0:
```
Comment (haojinIntel) [inline python/cloudtik/providers/_private/aws/config.py:1458]: route.target = "peering connection"

File: python/cloudtik/providers/_private/aws/config.py [hunk 2]
```diff
@@ -1404,12 +1429,95 @@ def _delete_vpc_endpoint_for_s3(ec2_client, vpc_id, workspace_name):
...
+    except Exception as e:
+        cli_logger.error("Failed to delete VPC peering connection. {}", str(e))
+        raise e
+    return
+
+
 def _create_vpc(config, ec2, ec2_client):
     cli_logger.print("Creating workspace VPC...")
     # create vpc
+    CidrBlock = '10.0.0.0/16'
```
Comment (haojinIntel) [inline python/cloudtik/providers/_private/aws/config.py:1513]: cidrBlock

**Comment by haojinIntel** [comment]
@Hi haifeng, I've refined and tested this PR. Note that we must update route tables for both current VPC and workspace VPC. Thanks!

**Comment by jerrychenhf** [comment]
I see. Thanks!

**Comment by jerrychenhf** [comment]
Thanks Hao for the contribution.

**Pull request merged**